UPGRADE-3.1.md

```diff
@@ -0,0 +1,26 @@
+UPGRADE FROM 3.0 to 3.1
+=======================
+
+DependencyInjection
+-------------------
+
+ * Using unsupported configuration keys in YAML configuration files has been deprecated and will raise an exception in Symfony 4.0.
+
+ * Using unsupported options to configure service aliases has been deprecated and will raise an exception in Symfony 4.0.
+
+Form
+----
+
+ * The `choices_as_values` option of the `ChoiceType` has been deprecated and will be removed in Symfony 4.0.
```

Comment:
In this line I miss the *"... use XXX instead"*  Or is it a removed feature without alternative?